apps/zk-explorer/utils/utils.ts

```diff
@@ -185,3 +149,19 @@ export function computeUserDiff(initial: User, updated: User): Partial<User> {
     (value, key) => !_.isEqual(value, initial[key as keyof User])
   );
 }
+
+export function gracefullyParseJson<T = unknown>(
+  jsonString: string
+): T | Error {
+  try {
+    return JSON.parse(jsonString);
+  } catch (possibleError) {
+    if (possibleError instanceof Error) {
+      return possibleError;
+    }
+
+    return new Error(
+      'Unknown error because the thrown object is not an instance of an error'
+    );
+  }
+}
```

Comment:
I believe we already have a similar function called `safeParseJson` in other dApps. It might be beneficial to consolidate these functions into a single one and place it in the `webb-ui-components`. This way, we can reuse it across various dApps, maintaining a single source of truth for the project.

Reply:
Do note that this will involve changes to `bridge-dapp`. Will need to test it somehow, to ensure nothing broke. I've made as few changes as possible to `bridge-dapp` to have it accustomed to the new consolidated `safeParseJson` function.

Reply:
Yep, sounds good to me 🆗